src/component/packaging/Packaging.jsx

```diff
@@ -1,16 +1,26 @@
 import React, { useEffect, useState } from "react";
 import axiosInstance from "../../axiosConfig";
+import { AgGridReact } from "ag-grid-react";
 import { Container } from "react-bootstrap";
 
+
 const Packaging = () => {
   const [packagingOrders, setPackagingOrders] = useState([]);
+  const [columnDefs] = useState([
```

Comment:
syntax wrong, write like const columnDefs = [{}]

Reply:
ok will do for now just merge in next pull it will be resolved .

Reply:
done you can merge .